tests/e2e/jest.config.js

```diff
@@ -14,6 +14,8 @@ module.exports = {
 	testPathIgnorePatterns: [
 		'.git',
 		'node_modules',
+		'tests/e2e/specs/block-editor',
```

Comment:
Why exclude the `block-editor`?

Reply:
Oh, because you said some tests are failing.

Reply:
If this is included, then no tests would be run at all, except for `tests/e2e/specs/amp-options.js`, right?

Can the `block-editor` test be fixed? What is the failure?

Reply:
Fixed the failing `block-editor` tests in https://github.com/ampproject/amp-wp/pull/4043/commits/1f842d3deff649af9494772cbb29d5db34f226fa.